### PR TITLE
validation of content, add_note method implemented

### PR DESCRIPTION
**Pull request by AndreyShvachko**

### CODE REVIEW

File: src/managers/note_manager.py [hunk 1]
```diff
@@ -9,13 +9,34 @@
...
+        Validates the note's title and content
 
+        """
+        if not note.title or len(note.title) < 5: #Number 5 is for example. Could be any number
+            print("Error: The title must not be empty and should have at least 5 characters.")
+            return False
+        if not note.content:
+            print("Error: The content must not be empty.")
+            return False
+        return True
```
Comment (AlexnderDoroshenko) [inline src/managers/note_manager.py:31]: Hi @AndreyShvachko !
Please add an empty line between the methods.

Comment (AndreyShvachko) [reply src/managers/note_manager.py:31]: Hi @AlexnderDoroshenko !
Done! Thank you for your note!

Comment (MHryhoriev) [reply src/managers/note_manager.py:31]: @AndreyShvachko hello!
How do you feel about us improving the logic a bit and making it more functional?
For example: Let's put a character limit in the argument with a default value:
def validate_note(self, note: Note, min_title_length: int = 5) -> None:
   if not note.title or len(note.title) < min_title_length:

Comment (AndreyShvachko) [reply src/managers/note_manager.py:31]: @MHryhoriev Hi!
Thanks for the note. Will do it

**Pull request merged**